src/test/java/com/mongodb/kafka/connect/source/schema/BsonDocumentToSchemaTest.java

```diff
@@ -198,20 +203,53 @@ void testArraysWithStructs() {
     Schema expected =
         SchemaBuilder.struct()
             .name(DEFAULT_FIELD_NAME)
-            .field("structs", createArray("structs", SIMPLE_STRUCT))
-            .field("structsEmpty", createArray("structsEmpty", SIMPLE_STRUCT))
-            .field("structsEmptyFirst", createArray("structsEmptyFirst", SIMPLE_STRUCT))
-            .field("structsNull", createArray("structsNull", SIMPLE_STRUCT))
-            .field("structsNullFirst", createArray("structsNullFirst", SIMPLE_STRUCT))
-            .field("structsOrdering", createArray("structsOrdering", SIMPLE_STRUCT))
+            .field("structs", createArray("structs"))
```

Comment:
Most tests had to also be changed now `assertSchemaEquals` actually asserts correctly :crying_cat_face: